src/components/pages/AddReviews/MentorAddReview.js

```diff
@@ -29,14 +29,15 @@ const MentorAddReview = () => {
     <Card
       title="Please fill out a review"
       style={{
-        width: '40rem',
+        width: 'auto', // changed from 40rem to auto
         margin: 'auto',
         display: 'flex',
         flexDirection: 'column',
       }}
       headStyle={{
         display: 'flex',
         justifyContent: 'center',
+        color: 'rgb(255, 255, 255, 0.85)', // added aug 18th
```

Comment:
not sure how useful THIS comment will be down the road. I can see it may have been helpful for you to track your progress, but now that it's working it could probably be cleaned up unless there's a reason I'm not understanding.

Reply:
Yes this can be cleaned up! Kicking myself for missing this and one other but we'll get it done!